exercises/nth-prime/example.jl

```diff
@@ -0,0 +1,43 @@
+function isPrime(x)
+	i = 2

```

Comment:
Perhaps move this down just in front of the loop to make it more readable. You could also consider a for-loop.

The internals of the example solution don't matter much, so I'll leave this up to you.